### PR TITLE
Initial translation to docutils

### PR DESCRIPTION
**Pull request by frank-at-adacore**

### CODE REVIEW

File: source/guidelines/definitions.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+This section contains terms and values used in the definitions of the rules set forth in this chapter.
+
+-------
+Level
+-------
+
+**Level** is the compliance level for the rule. Possible values are:
+
+   Mandatory
+      A non-compliance with a rule of the *Mandatory* recommendation
```
Comment (ButcherAdaCore) [inline source/guidelines/definitions.rst:15]: Non-compliance with a *Mandatory* recommendation level corresponds to a high risk of a software bug. There would need to be a good reason for non-conformity to a mandatory rule and, although it is accepted that exceptional cases may exist, any non-conformance should be accompanied by a clear technical explanation of the exceptional circumstance.

File: source/guidelines/definitions.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      level corresponds to a high risk of software bug.
+      As long as this non-conformity does not fall within the
+      exceptional cases stated here, its consideration is mandatory and
+      must systematically be the subject of a "technical fact".
+
+   Required
```
Comment (ButcherAdaCore) [inline source/guidelines/definitions.rst:21]: Non-compliance with a *Required* recommendation level corresponds to a medium to high risk of a software bug. Much like a *Mandatory* recommendation there would need to be a good reason for non-conformity to a required rule. Here it is accepted that more exceptional cases may exist, however, non-conformance should be accompanied by a clear technical explanation of the exceptional circumstance.

File: source/guidelines/definitions.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      not lead to systematically risk of bug on the software but is
+      considered as a risk significant deterioration in the level of
+      satisfaction of a quality characteristic, from the point of view
+      of the expected rules.
+      One or a set of *Required* level non-conformities may be the
+      subject of a "fact technical", since the benefit of remedying the
+      non-conformity or all the non-conformities considered contributes
+      significantly to improving the quality of the code.
+
+   Advisory
```
Comment (ButcherAdaCore) [inline source/guidelines/definitions.rst:32]: Failure to follow an *Advisory* recommendation does not necessarily result in a software bug; the risk of a direct correlation between non-conformance of an advisory rule and a software bug is low. Non-compliance with an advisory recommendation level does not require a supporting technical explanation, however, as the quality of the code may be impacted the reason for the non-conformance should be understood.

**Review by ButcherAdaCore — COMMENTED** [review]
Review of the "Definitions" sections

**Review by QuentinOchem — APPROVED** [review]
Looks good to me.

**Pull request merged**